### PR TITLE
docs: update web3 roadmap, open source projects, eth dev tooling

### PR DESCRIPTION
**Pull request by therealharpaljadeja**
## 👨‍💻 Changes proposed

- [x] Mentioning the speaker in the web3 roadmap video
- [x] Added celo-composer to open source projects
- [x] Added celo-composer to eth dev tooling
- [x]  Added Celo Developer Blog under blogs
- [x] Removed Duplicate Blogs and Web3 Roadmap section.

### CODE REVIEW

**Review by Atharv181 — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -38,17 +38,11 @@ This is a list of Free Web3 Resources links to learn Blockchain development.
...
 - [Become a Web 3 & Blockchain Developer in 2022 | Practical Step by Step Solidity and Web3 Roadmap](https://www.youtube.com/watch?v=aVQJGr2J8io) by Javascript Mastery
 - [A simple guide to the web3 developer stack](https://www.coinbase.com/blog/a-simple-guide-to-the-web3-developer-stack) by Coinbase
-- [How to Become a Blockchain Developer? Complete web3 Developer Roadmap](https://www.youtube.com/watch?v=q54j35z3fPQ) by Ishan Sharma
+- [How to Become a Blockchain Developer? Complete web3 Developer Roadmap](https://www.youtube.com/watch?v=q54j35z3fPQ) by [Harpalsinh Jadeja](https://twitter.com/HarpalJadeja11) on Ishan Sharma's Youtube Channel
 - [Complete Blockchain Development Roadmap 2022 🔥| Project Ideas | Solidity | Web3js | Truffle](https://www.youtube.com/watch?v=36ubP5Ydh-A) by Nishant Chahar
 - [How to become a Blockchain Developer in 2022? | 🔥 Complete Roadmap + Salary + Tech Stack](https://www.youtube.com/watch?v=uULy2rc6YDc) by Apna College
 - [How to get started with Web3 Development - Become Blockchain Developer in 2022 Complete Roadmap](https://www.youtube.com/watch?v=XRIpjWGbGmA) by Coding Money
 
-## Web3 Roadmaps (Hindi/ Urdu)
-
```
Comment (FrancescoXX) [inline README.md:47]: please put this back

File: README.md [hunk 2]
```diff
@@ -38,17 +38,11 @@ This is a list of Free Web3 Resources links to learn Blockchain development.
...
-- [Complete Blockchain Development Roadmap 🔥 What is Blockchain and how to get started? Salary & Course](https://youtu.be/n_5oeZWbseY)
-
-## Blogs
```
Comment (FrancescoXX) [inline README.md:50]: please put this back

File: README.md [hunk 3]
```diff
@@ -38,17 +38,11 @@ This is a list of Free Web3 Resources links to learn Blockchain development.
 
 - [Become a Web 3 & Blockchain Developer in 2022 | Practical Step by Step Solidity and Web3 Roadmap](https://www.youtube.com/watch?v=aVQJGr2J8io) by Javascript Mastery
 - [A simple guide to the web3 developer stack](https://www.coinbase.com/blog/a-simple-guide-to-the-web3-developer-stack) by Coinbase
-- [How to Become a Blockchain Developer? Complete web3 Developer Roadmap](https://www.youtube.com/watch?v=q54j35z3fPQ) by Ishan Sharma
+- [How to Become a Blockchain Developer? Complete web3 Developer Roadmap](https://www.youtube.com/watch?v=q54j35z3fPQ) by [Harpalsinh Jadeja](https://twitter.com/HarpalJadeja11) on Ishan Sharma's Youtube Channel
 - [Complete Blockchain Development Roadmap 2022 🔥| Project Ideas | Solidity | Web3js | Truffle](https://www.youtube.com/watch?v=36ubP5Ydh-A) by Nishant Chahar
 - [How to become a Blockchain Developer in 2022? | 🔥 Complete Roadmap + Salary + Tech Stack](https://www.youtube.com/watch?v=uULy2rc6YDc) by Apna College
 - [How to get started with Web3 Development - Become Blockchain Developer in 2022 Complete Roadmap](https://www.youtube.com/watch?v=XRIpjWGbGmA) by Coding Money
 
-## Web3 Roadmaps (Hindi/ Urdu)
```
Comment (FrancescoXX) [inline README.md:46]: please put this back

**Review by FrancescoXX — CHANGES_REQUESTED** [review]

**Pull request merged**